### ISSUE TITLE
H.264 encoding

### ISSUE THREAD

**Issue by funkyeah** [MAVProxyUser/RyzeTelloFirmware#1]
Hey, ran across this repo trying to figure out how the Tello works. So are you do you believe the Movidius does h.264 encoding, streams that to the WiFi processor, and also does the motor control?

**Comment by ArcadeHustle** [comment]
yes I believe the Movidius is at the core, including doing the PWM. They are using non public APIs from movidius. You have to be a bit time player to get access to the API at that level.
